src/components/pill-selector-dropdown/PillSelectorDropdownField.js

```diff
@@ -14,19 +14,21 @@ type Props = FieldProps & {
     /** CSS class for the component. */
     className?: string,
     /** Given selected values and input text, returns a list of filtered options. Defaults to defaultDropdownFilter. */
-    dropdownFilter?: (options: Array<Option>, selectedValues: Array<Option>, inputText: string) => Array<Option>,
+    dropdownFilter: (options: Array<Option>, selectedValues: Array<Option>, inputText: string) => Array<Option>,
```

Comment:
I think removing the `?` might make flow complain about a missing prop if the consumer doesn't provide a filter.

Reply:
Its part of defaultProps, so flow should be happy

Reply:
Oh ok, cool. Didn't know that flow used the `defaultProps`